go/test/endtoend/vreplication/vdiff2_test.go

```diff
@@ -242,6 +249,64 @@ func testCLIErrors(t *testing.T, ksWorkflow, cells string) {
 	})
 }
 
+// testCLIFlagHandling tests that the vtctldclient CLI flags are handled correctly
+// from vtctldclient->vtctld->vttablet->mysqld.
+func testCLIFlagHandling(t *testing.T, targetKs, workflowName string, cell *Cell) {
+	// Keys are in the tabletmanagerdata.VDiff*Options proto message definitions.
+	coreOpts := map[string]string{
+		"max_rows":                  "999",
+		"max_extra_rows_to_compare": "777",
```

Comment:
It will be interesting to see what happens if someone does provide a negative number for any of these numeric flags.